### PR TITLE
Add snap install instructions

### PR DESCRIPTION
**Pull request by flexiondotorg**
This pull request adds installation instructions for users of [snap enabled Linux distributions](https://snapcraft.io/docs/core/install).

### CODE REVIEW

File: pages/docs/tutorials/command-line.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ Every release ships with a standalone version of the compiler. We can download i
 #### Manual Install
 Unzip the standalone compiler into a directory and optionally add the `bin` directory to the system path. The `bin` directory contains the scripts needed to compile and run Kotlin on Windows, OS X and Linux.
 
+#### Linux
+
+You can install the auto-updating snap of the Kotlin compiler on [Ubuntu and other snap supported Linux distributions](https://snapcraft.io/docs/core/install) with:
+
+```
+   $ snap install kotlin --classic
```
Comment (ARostovsky) [inline pages/docs/tutorials/command-line.md]: sudo?

Comment (flexiondotorg) [reply pages/docs/tutorials/command-line.md]: @ARostovsky PolicyKit will request elevated permissions.

**Comment by belovrv** [comment]
Thanks!
Just approved yet another PR about it.

**Comment by h0tk3y** [comment]
Since another PR with equivalent changes has already been merged (#1018), I'm closing this one.

**Pull request closed without merging**